examples/debian/aci-debian-cassandra/runlevels/build/10.install.sh

```diff
@@ -10,12 +10,15 @@ deb-src http://www.apache.org/dist/cassandra/debian 30x main
 deb http://ftp.debian.org/debian jessie-backports main
 EOF
 
+gpg --keyserver pool.sks-keyservers.net --recv-keys F758CE318D77295D || \
```

Comment:
nice, can you put that in another PR, it's not linked.

Also can you squash your commits, 13 commits for a single feature too much